datajunction-server/datajunction_server/internal/nodes.py

```diff
@@ -295,7 +295,7 @@ def create_cube_node_revision(  # pylint: disable=too-many-locals
     dimension_attribute = session.exec(
         select(AttributeType).where(AttributeType.name == "dimension"),
     ).one()
-    dimensions_set = {dim.rsplit(SEPARATOR, 1)[1] for dim in data.dimensions}
+    dimensions_set = {dim.replace(SEPARATOR, "_DOT_") for dim in data.dimensions}
```

Comment:
If its not too hard lets define this in constants. It will be easier to find later.